packages/orbit-components/src/primitives/TooltipPrimitive/__tests__/index.test.js

```diff
@@ -5,7 +5,9 @@ import { render } from "@testing-library/react";
 import Tooltip from "..";
 import Airplane from "../../../icons/Airplane";
 
-jest.mock("../../../utils/randomID", () => name => `${name}50000-id-50000`);
+jest.mock("../../../hooks/useRandomId", () => ({
+  useRandomId: () => 1,
+}));
```

Comment:
Mocking `useRandomId` is unnecessary, I tested locally and works without.

Reply:
it was failing on CI with screenshots, so i thought the reason is in the changing id.